components/03-organisms/menu/primary-nav/_primary-nav.scss

```diff
@@ -0,0 +1,201 @@
+@use '../../../00-tokens/tokens';
+@use '../../../01-atoms/atoms';
+
+// @TODO: Move these to a token
+:root {
+  --color-muted: var(--color-gray-600);
+  --color-navigation-border: var(--color-blue-yale);
+  --color-navigation-expanded-item: var(--color-basics-brown-gray);
+}
+
+@mixin primary-nav-item-level-0 {
+  font: var(--font-style-nav-primary-0);
+  color: var(--color-heading);
+  text-align: left;
+
+  &[aria-expanded='true'] {
+    color: var(--color-navigation-expanded-item);
+  }
+
+  @media (max-width: tokens.$break-mobile-max) {
+    padding: var(--size-spacing-2) var(--size-spacing-5) var(--size-spacing-2);
+    margin-block: var(--size-spacing-3);
+  }
+
+  @media (min-width: tokens.$break-mobile) {
+    position: relative;
+    padding: var(--size-spacing-2) var(--size-spacing-5) var(--size-spacing-5);
+    border-bottom: var(--border-thickness-1) solid transparent;
+    border-top: var(--border-thickness-1) solid transparent;
+    border-right: var(--border-thickness-1) solid transparent;
+    border-left: var(--border-thickness-1) solid transparent;
+
+    &[aria-expanded='true'] {
+      z-index: 1;
+      border-color: var(--color-navigation-border);
+      border-bottom-color: var(--color-background);
+    }
+  }
+}
+
+@mixin primary-nav-item-level-1 {
```

Comment:
Not a blocker for merging but these mixins are only used once and are not very big.

Reply:
Done